DEPENDENCIES.rst

```diff
@@ -106,6 +114,14 @@ and (optionally):
 #. libblas-dev
 #. liblapack-dev
 #. libgoogle-perftools-dev
+#. python3
+#. python3-dev
+#. python3-tables
+#. python3-pandas
+#. python3-numpy
+#. python3-nose
+#. python3-jinja2
+#. cython3
 
```

Comment:
Ubuntu 16 has python2 as the default, so this set of dependencies will not work.  I have demonstrated that it will not work.

If we remove the `3`, it might (??) work on all systems because it will get the appropriate version.